mitodl/hugo-course-publisher
# anchor shortcode content

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/ocw-to-hugo/issues/75

#### What's this PR do?
Sets the inner HTML content of the generated `a` tag to `.Inner` in the `anchor` shortcode template.

#### How should this be manually tested?
Render some markdown on [this](https://github.com/mitodl/ocw-to-hugo/pull/81) branch of `ocw-to-hugo` and then run the site build against it.  Anchor tags with `name` properties should now have their inner HTML content properly displayed.

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/12089658/93791294-0c9d7000-fc02-11ea-963a-46b4234a8283.png)
